lib/engine-addon.js

```diff
@@ -126,9 +129,17 @@ var buildVendorCSSTree = memoize(function buildVendorCSSTree(vendorTree) {
 
 var buildEngineJSTree = memoize(function buildEngineJSTree() {
   var engineSourceTree;
-  var treePath = path.resolve(this.root, this.treePaths['addon']);
-  if (existsSync(treePath)) {
+  var treePath;
+  var addonTree = this.options.trees.addon;
```

Comment:
Ah, okay I see. Since a string value is the default here I didn't consider setting it to a non-string value.